### ISSUE TITLE
`NeuronTree.smooth_neurites` is broken

### ISSUE THREAD

**Issue by jnsbck** [berenslab/MorphoPy#124]
`NeuronTree.smooth_neurites` is broken on most recent commit in `main` and latest release (with `networkx v2.8`).

Here is a minimally reproducable example:
```python
from morphopy.computation import file_manager as fm

neuron = fm.load_swc_file("data/C4.swc")
neuron = neuron.smooth_neurites(dim=1, window_size=21)
```
The following Exception is thrown:

![image](https://user-images.githubusercontent.com/65561470/213713427-08ae5890-b6bc-4f54-adbb-3f1faa4562df.png)

**Comment by jnsbck** [comment]
@huangziwei maybe we can look into it together.

**Comment by huangziwei** [comment]
fixed by #125
